--- libcmis-0.3.0.orig/Makefile.am
+++ libcmis-0.3.0/Makefile.am
@@ -13,7 +13,7 @@ pkgconfigdir = $(libdir)/pkgconfig
 
 if WITH_LIBCMIS_MAN
 cmis-client.1: doc/cmis-client.xml
-	$(DOCBOOK2MAN) $<
+	$(DOCBOOK2MAN) $< > $@
 
 install-man: cmis-client.1
 	mkdir -p $(DESTDIR)$(mandir)/man1/
--- libcmis-0.3.0.orig/Makefile.in
+++ libcmis-0.3.0/Makefile.in
@@ -869,7 +869,7 @@ libcmis-c-@LIBCMIS_API_VERSION@.pc: libc
 		@cp -f $< $@
 
 @WITH_LIBCMIS_MAN_TRUE@cmis-client.1: doc/cmis-client.xml
-@WITH_LIBCMIS_MAN_TRUE@	$(DOCBOOK2MAN) $<
+@WITH_LIBCMIS_MAN_TRUE@	$(DOCBOOK2MAN) $< > $@
 
 @WITH_LIBCMIS_MAN_TRUE@install-man: cmis-client.1
 @WITH_LIBCMIS_MAN_TRUE@	mkdir -p $(DESTDIR)$(mandir)/man1/
